src/index.html

```diff
@@ -513,16 +515,23 @@ <h2 class="h4">Options</h2>
         </section>
         <div class="mt-5">
           <button type="submit" class="btn btn-outline-primary btn-block btn-lg mt-3 js-btn-customize">
-            <span class="bx bx-cog" aria-hidden="true"></span> Build
+            <svg class="icon-build" viewBox="0 0 32 32" width="32" height="32" fill="none" stroke="currentcolor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2">
+              <path d="M13 2 L13 6 11 7 8 4 4 8 7 11 6 13 2 13 2 19 6 19 7 21 4 24 8 28 11 25 13 26 13 30 19 30 19 26 21 25 24 28 28 24 25 21 26 19 30 19 30 13 26 13 25 11 28 8 24 4 21 7 19 6 19 2 Z" />
+              <circle cx="16" cy="16" r="4" />
+            </svg> 
+            <span class="align-middle">Build</span>
           </button>
         </div>
       </form>
     </div>
-
     <footer class="bg-light p-3 mt-4">
       <div class="container">
         <p class="text-muted text-center mb-0">
-          Made with <span class="bx bxs-heart text-danger" aria-hidden="true"></span><span class="sr-only">love</span> by <a href="https://github.com/Johann-S">Johann-S</a>
+          Made with 
+          <svg class="icon text-danger" viewBox="0 0 32 32" width="32" height="32" fill="none" stroke="currentcolor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2">
```

Comment:
Are you sure about using the outline version? Note that the Unicode characters also works.

Also, if you go with the SVG, I'd make it a lot smaller like 16x16.

Reply:
Not sure about the outline version 😟 I used: https://github.com/danklammer/bytesize-icons.

Do not hesitate to reduce our SVG 👍

Reply:
Yeah, that doesn't have a solid heart icon.

Reply:
If you have a solid one do not hesitate 👍

Reply:
We could always use the unicode symbol.

Reply:
But anyway that's nitpicking :)